src/app/pages/products/components/filters/filter-panel/filter-panel.component.ts

```diff
@@ -32,21 +32,14 @@ export class FilterPanelComponent implements OnInit {
     private cdr: ChangeDetectorRef,
   ) {}
 
-  ngOnInit() {
-    this.filterFacadeService.getFilterValue()
+  async ngOnInit(): Promise<void> {
+    await firstValueFrom(this.filterFacadeService.getFilterDefinitions());
```

Comment:
The row is duplicated, do we need it?

Reply:
We don't need the duplicated row, you were right, I refactored!